CompuCell3D/core/CompuCell3D/plugins/FocalPointPlasticity/FocalPointPlasticityLinks.h

```diff
@@ -174,6 +181,17 @@ namespace CompuCell3D {
 		CellG* getObj0() { return initiator; }
 		// Get second object
 		CellG* getObj1() { return initiated; }
+
+		// Python support
+
+		PyObject *pyAttrib;
+
+		PyObject *getPyAttrib() {
+#ifdef SWIGPYTHON
```

Comment:
you might need to be careful here when it to memory usage. I dont remember exactly the issue but I think I was able to create memory leaks. I think there might be some explanation of this in the comments of PyAttributeAdder